aces/Loris
# [Timepoint/Create Timepoint] Add timepoint language feature

## Brief summary of changes
This PR adds language feature to timepoints. 

#### Testing instructions (if applicable)

1. Run patches in this PR + make sure you have changes that were merged yesterday from #4492
2. Create a new timepoint and choose a language.
3. You can add french test names  to `test_names_fr` so the instrument list is translated according to the language of the session.
